src/types.js

```diff
@@ -42,7 +50,7 @@ export type PoliticianType = {
   districtNumber: number,
   party: PartyId,
   district: DistrictType,
-  promises: Array<PromiseType>,
+  promises: Array<number>,
```

Comment:
I've been trying to write `SomeTypeId` in these cases.

I realize it's more terse, but it instantly tells me what the data-structure I'm looking at is.

If you don't agree with that, then I'll stop doing it. No use in having a convention if not everybody follows it